pkg/operator/target_config_reconciler.go

```diff
@@ -1207,12 +1405,13 @@ func (c *TargetConfigReconciler) manageDeployment(descheduler *deschedulerv1.Kub
 	}
 
 	images := map[string]string{
-		"${IMAGE}": c.targetImagePullSpec,
+		"${IMAGE}": targetImagePullSpec,
 	}
 	for i := range required.Spec.Template.Spec.Containers {
 		for pat, img := range images {
 			if required.Spec.Template.Spec.Containers[i].Image == pat {
 				required.Spec.Template.Spec.Containers[i].Image = img
+				required.Spec.Template.Spec.Containers[i].ImagePullPolicy = v1.PullAlways // TODO: remove me
```

Comment:
Maybe you forgot to remove this line (or the comment).

Reply:
This one is still valid

Reply:
Done